### DISCUSSION TITLE
Create A Repository Tab Not Transferring Files

### DISCUSSION THREAD

**Discussion by aimastercloser** [community/community#148208] [New to GitHub]
It creates the repo but the files don't appear... Has anyone else come across this issue when using this option inside Bolt.New?
![2024-12-30_14-59-27](https://github.com/user-attachments/assets/efdc79d4-176a-462d-862b-d89a0489a3ff)

**Comment by nishanmudalige** [comment]
When you click the GitHub button on the top right, do you get a prompt to `Confirm GitHub Upload` with the option of typing a message?

**Reply by aimastercloser** [reply]
Thank you for your response! The prompt does not appear... I found a solution using a chrome extension called...Bolt to Github.
